src/node.h

```diff
@@ -251,6 +251,10 @@ NODE_EXTERN Environment* CreateEnvironment(IsolateData* isolate_data,
 NODE_EXTERN void LoadEnvironment(Environment* env);
 NODE_EXTERN void FreeEnvironment(Environment* env);
 
+// This returns the MultiIsolatePlatform used by node main.
+// If NODE_USE_V8_PLATFORM is not defined, it will return nullptr.
```

Comment:
Maybe specific that the macro needs to have been defined when Node.js was built, not when the addon is being built?

Reply:
Also, what's "node main"?  This comment could be expanded a little.